docs/dependencies/aliases.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Package aliases
+---
+
+Now that you have [published a shared dependency](/docs/dependencies/npm/) to Eik and seen how to update it, it's time to set up an alias.
```

Comment:
```suggestion
Now that you have [published package](/docs/dependencies/npm/) to Eik and seen how to update it, it's time to set up an alias.
```
I think "shared dependency" is a bit alien, and we genereally use package elsewhere in the docs.

Reply:
I think I wanted to distinguish "shared dependency"/npm from "application package"/pkg. But we can use "package" here, no worries.

Reply:
Hmm, feels a bit clunky in places

> Like how the `npm` package type is designed for shared packages, `map` is designed to hold your import maps.

versus

> Like how the `npm` package type is designed for shared dependencies, `map` is designed to hold your import maps.

The "package" term is a bit loaded in Eik 😅

Reply:
yeah, it all get's a bit messy 😄 however I'm not sure using an unfamiliar term makes it easier, as it leaves the reader to figure out what is ment by it.

it might be ok to leaved it... as "shared packages" is kind of implicit in this context to it can end up like this:
> Like how the `npm` package type is designed for npm packages, map is designed to hold your import maps.